desktop/apps/auctions2/reducers/appReducer.js

```diff
@@ -1,17 +1,39 @@
 import * as types from 'desktop/apps/auctions2/actions/appActions'
 import u from 'updeep'
+import { filter, reject } from 'underscore'
 
 export const initialState = {
-  isMobile: false
+  isMobile: false,
+  auctions: {
+    live: [],
+    timed: []
+  },
+  isFetchingAuctions: false
 }
 
 export default function appReducer (state = initialState, action = {}) {
   switch (action.type) {
-    case types.TEST_ACTION: {
-      console.log('Hello test action!')
-
+    case types.GET_AUCTIONS_FAILURE: {
+      return u({
+        isFetchingAuctions: false
+      }, state)
+    }
+    case types.GET_AUCTIONS_REQUEST: {
+      return u({
+        isFetchingAuctions: true
+      }, state)
+    }
+    case types.GET_AUCTIONS_SUCCESS: {
+      const auctions = action.payload.auctions
+      const isLive = function(auction){return auction.live_start_at ? true : false}
```

Comment:
Can use ES6 `=>` fn and rely on truthiness (either it exists, which is truthy, or not, which returns `undefined` which in turn is evaluated to `false`):

```
const isLive = (auction) => auction.live_start_at 
```